libs/wingsdk/src/std/bool.ts

```diff
@@ -7,13 +8,22 @@ export class Boolean {
   /**
    * Parse a boolean from Json.
    *
-   * @macro ((args) => { if (typeof args !== "boolean") {throw new Error("unable to parse " + typeof args + " " + args + " as a boolean")}; return JSON.parse(JSON.stringify(args)) })($args$)
-   *
    * @param json to parse boolean from.
    * @returns a boolean.
    */
   public static fromJson(json: Json): boolean {
-    json;
-    throw new Error("Macro");
+    if (typeof json !== "boolean") {
+      throw new Error(
+        "unable to parse " + typeof json + " " + json + " as a boolean"
+      );
+    }
+    return JSON.parse(JSON.stringify(json));
+  }
```

Comment:
Does this mean we can just get rid of macros? 👀  Big if true

Reply:
I wouldn't get rid of them. They are a Good Thing™️ and required in order to "wrap" our primitives with Wing APIs. They are also a runtime optimization.